### DISCUSSION TITLE
Is this js file still working?

### DISCUSSION THREAD

**Discussion by NCLnclNCL** [kaliiiiiiiiii/Selenium-Profiles#105] [Q&A] [answered]
https://github.com/kaliiiiiiiiii/Selenium-Profiles/blob/master/dev_stuff/devtools_anti_detector.js

**Comment by ttlns** [comment, marked as answer]
@NCLnclNCL Please only open an issue when there's a bug! For questions, refer to Q&A at discussions.

To answer your question: No, that file isn't used anywhere. For passing devtools detection, you'll have to use `selenium-driverless`
